test/dependency_injection_test.rb

```diff
@@ -2,11 +2,15 @@
 
 class TestDependencyOne
   attr_reader :arg
-  def initialize(arg = nil); @arg = arg; end
+  def initialize(arg = nil);
```

Comment:
are the semi colons still needed?

Reply:
Nope, changed for these ones now. There's another cop for it, that finds more. ;)